### ISSUE TITLE
Community / Collection statistics should include aggregate statistics for their Items

### ISSUE THREAD

**Issue by tdonohue** [DSpace/DSpace#8572]
**Is your feature request related to a problem? Please describe.**
This was brought up in a DCAT discussion today, and was supported by several DCAT members.

Currently, in DSpace 7, the (Solr) Statistics pages for Communities and Collections just displays statistics for *views* of the Community/Collection homepage.  While this is useful, DCAT members agreed it'd be useful to also show aggregated statistics for all Items in that Community/Collection.  Some examples of aggregated statistics include:
* total bitstream downloads
* most downloaded (top 10?) bitstreams
* total Item accesses
* most accessed (top 10?) Items.

Sidenote: It's also odd that the historical view statistics for Communities/Collections only show the last 7 months.  7 months is an odd timeframe, and likely should be updated to be either 6 or 12 months, or maybe a way to select the number of months to show?

**Comment by mwoodiupui** [comment]
I've never poked around in the built-in statistics, but I'd guess that 7 months is to cover the difference between "6 calendar months" and "6 months ending now".

Not disagreeing with the meat of this issue, but I would say that, if one is more than casually interested in statistical analysis of DSpace usage, one should export the cases into R or SAS or SPSS or Stata or any of a dozen others, and get the abstractions that one wants.  We should have simple statistics built in, but **strongly** resist the temptation to add more and more and more.  (I've said this before, and I'll continue to say it at intervals, because it's so easy to succumb to this particular temptation.  Everybody wants "statistics," but everybody means something different....)

**Comment by mwoodiupui** [comment]
I've never poked around in the built-in statistics, but I'd guess that 7 months is to cover the difference between "6 calendar months" and "6 months ending now".

Not disagreeing with the meat of this issue, but I would say that, if one is more than casually interested in statistical analysis of DSpace usage, one should export the cases into R or SAS or SPSS or Stata or any of a dozen others, and get the abstractions that one wants.  We should have simple statistics built in, but **strongly** resist the temptation to add more and more and more.  (I've said this before, and I'll continue to say it at intervals, because it's so easy to succumb to this particular temptation.  Everybody wants "statistics," but everybody means something different....)

And now I see #8573.  I'll continue my ranting there.

**Comment by tdonohue** [comment]
@mwoodiupui : Agreed overall... one of the points made in the DCAT meeting though was that external tools often don't "understand" DSpace's Community <-> Collection <-> Item <-> Bitstream structure.  So, showing statistics based on this structure is something that can be done relatively easily _with DSpace_, but may not be as easy in external analysis tools (unless you can feed those tools somehow with pre-aggregated data). 

This is where the idea of having _basic_ "aggregated stats" comes in... it's not (easily) possible for Google Analytics (or other external tools) to easily tell you how many downloads occurred in Collection A vs Collection B.  But, DSpace's Solr Statistics could do that relatively easily (though its numbers may not always be as accurate as a system which has streamlined filtering out bots)

**Comment by eulereadgbe** [comment]
It would also be nice if there is an option to select the date start and date end for each aggregated statistic. Our use case requires us to report the number of downloads, and access each year or for a specific time frame.

**Comment by mwoodiupui** [comment]
I wonder whether the DSpace-CRIS merge will affect this.

**Comment by tdonohue** [comment]
@mwoodiupui : Yes, I believe DSpace-CRIS has some better aggregation / custom report options available, and those will be moved to DSpace in #11782.  

That said, I'm not sure whether DSpace-CRIS has Community/Collection aggregation out of the box or not.  Nonetheless, I'm flagging this as blocked by #11782 because we should revisit this ticket once that one is resolved & determine whether any further work is needed (or not).
